examples/contour/02-service-envoy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+ name: envoy
+ namespace: heptio-contour
+ annotations:
+   service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
is this a change from the previous non split example? I thought it was an elb or alb.

Reply:
Yes, I've picked up the wrong one as the base here. Thanks.